akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -31,6 +31,7 @@ object Source {
   val factory: SourceCreate = new SourceCreate {}
 
   /** Adapt [[scaladsl.Source]] for use within JavaDSL */
+  // FIXME: is this needed now?
```

Comment:
isn't it now enough to just call wrap, since that works with any Graph[SourceShape]? Why specialize it to scaladsl.Source?